### PR TITLE
Sponsored CAPI component restyling

### PR DESCRIPTION
**Pull request by uplne**
After:
screen shot 2016-01-25 at 10 56 58

It is behind the switch so till the switch is off it looks like old CAPI component.
It uses @jbreckmckye amazing work on badges position so it works like a magic. I just needed to add '.commercial--dfp-single' into the equation.

TODO: after launch we will have to do massive clean up of redundant code which is behind the switch

CC @jbreckmckye @regiskuckaertz

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
Shouldn't the colon after `paid for by` disappear?

**Comment by uplne** [comment]
@regiskuckaertz nice catch. Fixed.

File: commercial/app/views/contentapi/item.scala.html [hunk 1]
```diff
@@ -1,8 +1,83 @@
...
+    @if(NewCommercialContent.isSwitchedOn) {
+    <div class="commercial commercial--dfp-single paid-content--advertisement-feature commercial--tone-paidfor" data-link-name="merchandising | capi | single @optOmnitureId">
+        <div class="commercial__inner">
+            <div class="commercial__header commercial--paidfor">
+                <div class="paidfor-meta">
+                    <span class="paidfor-meta__label">Paid Content</span>
+                    <div class="paidfor-label paidfor-meta__more has-popup">
+                        <button class="u-button-reset paidfor-label__btn popup__toggle" data-toggle="paidfor-popup--capi-component">About @inlineSvg("arrow-down", "icon")</button>
+                        <div class="popup is-off paidfor-popup paidfor-popup--capi-component">
+                            <h3 class="paidfor-popup__title">Paid stories are paid for and approved by an advertiser and created by the Guardian Labs commercial editorial team</h3>
```
Comment (regiskuckaertz) [inline commercial/app/views/contentapi/item.scala.html]: The popup content has been updated, you should check it out (text+link)

Comment (uplne) [inline commercial/app/views/contentapi/item.scala.html]: @regiskuckaertz I've updated the text and created card so we should look at it one more time to possibly create fragment for this as it is across multiple components as you said.

**Comment by regiskuckaertz** [comment]
:+1: once the link is fixed yeah!

**Pull request merged**